hug/api.py

```diff
@@ -209,18 +182,6 @@ def documentation(self, base_url=None, api_version=None):
         documentation['handlers'] = version_dict
         return documentation
 
-    def serve(self, port=8000, no_documentation=False):
```

Comment:
unfortunately, this pull request substantially changes the API behind how hug servers are started manually - a function used by many people. This makes it hard for me to accept the pull request as-is